README.md

```diff
@@ -11,7 +11,7 @@
 
 - [Key Features](#key-features)
 
-🚀 [Live Demo](#live-demo)
+
```

Comment:
- [OPTIONAL] Kindly, it would be better if your ` Contributing` section had a link to the issue page, which will allow other developers to contribute to your project or create an issue if needed. let's add this. ✍🏻